Cargo.toml

```diff
@@ -58,13 +59,17 @@ openssl = { version = "0.10.48", features = ["vendored"] }
 
 nydus-api = { version = "0.2.2", path = "api", features = ["handler"] }
 nydus-rafs = { version = "0.2.2", path = "rafs", features = ["builder"] }
-nydus-service = { version = "0.2.0", path = "service", features = ["block-device"] }
+nydus-service = { version = "0.2.0", path = "service", features = [
```

Comment:
Unrelated change？

Reply:
This is the change made by the first two commits from **#1305**. I don't understand this part, I work on the third commit,  just calling the subcommands `nydus-image` chunkdict save ” implemented by the first two commits.